Packs/CommonScripts/.pack-ignore

```diff
@@ -106,6 +106,12 @@ ignore=RN114
 [file:DBotUpdateLogoURLPhishing.yml]
 ignore=SC106,BA124
 
+[file:context_entry_list_of_dicts_complex.json]
+ignore=BA102
+
+[file:expected_list_of_dicts_complex.json]
+ignore=BA102
+
```

Comment:
if you put it under a `test_data` dir it's not necessary

Reply:
The name of test_data dir is SetGridField_test, should we change it?

Reply:
yes